test/e2e/set/scyllacluster/scyllacluster_scaling.go

```diff
@@ -42,8 +41,12 @@ var _ = g.Describe("ScyllaCluster", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
-		hosts := getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
+
+		hosts, hostIDs, err := utils.GetBroadcastRPCAddressesAndUUIDs(ctx, f.KubeClient().CoreV1(), sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(hosts).To(o.HaveLen(1))
+		o.Expect(hostIDs).To(o.HaveLen(1))
```

Comment:
tiny nit: I'd technically rather assert `hostIDs` and `hosts` length for equality (globally)

Reply:
equal length follows from the `GetBroadcastRPCAddressesAndUUIDs` function, i.e. otherwise you'd receive an error. I suppose I can remove the explicit check for both and only check it for one, since the other follows implicitly